escher/gl/unique_shader.cc

```diff
@@ -0,0 +1,61 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "escher/gl/unique_shader.h"
+
+#include <iostream>
+
+namespace escher {
+namespace {
+UniqueShader CompileShader(GLenum type,
+                           GLsizei count,
+                           const GLchar** contents,
+                           const GLint* lengths) {
+  UniqueShader shader;
+  shader.Reset(glCreateShader(type));
+  glShaderSource(shader.id(), count, contents, lengths);
+  glCompileShader(shader.id());
+
+  GLint compile_status;
+  glGetShaderiv(shader.id(), GL_COMPILE_STATUS, &compile_status);
+  if (compile_status == GL_TRUE)
```

Comment:
nit: I prefer to avoid comparisons with true and false and just let the bool operator do its job.